frontend/src/types/enums.ts

```diff
@@ -81,3 +76,11 @@ export enum ToastType {
   WARNING = "warning",
   INFO = "info",
 }
+
+export enum SocketEvent {
+  CONNECT = "connect",
+  DISCONNECT = "disconnect",
+  JOIN_ROOM = "join_room",
+  CODE_CHANGE = "code_change",
+  CODE_UPDATE = "code_update",
```

Comment:
Will it cause any issues if I include matching socket event here? if ok, then i'll add it in 😊

Reply:
No issues! Pls do HAHAHA